rackerlabs/nexus-control
# Logo update

update logo to the new Rackspace Technology version for both support and developer

Testing instructions:

Pull down the PR locally and test both support and developer. Just `script/add-assets PATH-TO-NEXUS-CONTROL` is all you need. The logo in the header should be the Rackspace Technology logo.
